content/250-postgres/500-backups.mdx

```diff
@@ -6,7 +6,9 @@ tocDepth: 3
 toc: true
 ---
 
-Prisma Postgres takes full snapshots of your database every 24 hours to back up your data. If you need to access a database backup, [reach out to our Support team](mailto:support@prisma.io) from the email address associated with your Prisma Data Platform account. Doing so helps us verify your account quickly and expedite the restoration process.
+Prisma Postgres takes full snapshots of your database every 24 hours to back up your data. Automated daily backups are not available on the Starter plan. They are available exclusively on Pro, Business, and Enterprise plans. For more details about backup availability and plan-specific features, visit our [pricing page](https://www.prisma.io/pricing).
+
+If you need to access a database backup, [reach out to our Support team](mailto:support@prisma.io) from the email address associated with your Prisma Data Platform account. Doing so helps us verify your account quickly and expedite the restoration process.
 
 Please note that any database changes or events that occurred after the most recent snapshot may not be restored.
```

Comment:
how about this?

```suggestion
Prisma Postgres takes full snapshots of your database every 24 hours to back up your data. 

If you are on a Pro, Business, or Enterprise plan, you can [reach out to our Support team](mailto:support@prisma.io) to get access to a past snapshot. Backups are not available on the Starter plan. Please reach out from the email address associated with your Prisma Console account to help us verify your account quickly and expedite the restoration process.

Please note that any database changes or events that occurred after the most recent snapshot may not be restored.

For more details about backup availability and plan-specific features, visit our [pricing page](https://www.prisma.io/pricing).
```

Reply:
Looks good. Thanks for the suggestion! 👍